### PR TITLE
Adding the code for the add Contact to campaign

### PR DESCRIPTION
**Pull request by ishrartx**
Adding the code for the add Contact to campaign

### CODE REVIEW

File: src/test/resources/features/SalesMarketingCRM.feature [hunk 1]
```diff
@@ -9,17 +9,28 @@ Feature: Salesforce Marketing CRM
...
-    Then user clicks on Campaigns Tab
-    Then user selects the edit button
-    Then user selects the Delete button
-    Then user create new Campaign
-    Then user clicks on Campaigns Tab
-    Then user selects the parent Campaign
-    And user clicks on the Campaign
-    Then user create contact in Campaign
-    Then user Add the leads in campaign
+#    Then user clicks on Campaigns Tab
```
Comment (snaphray) [inline src/test/resources/features/SalesMarketingCRM.feature]: Please uncomment the commented lines if they are required

File: src/test/java/utilities/KeywordUtil.java [hunk 2]
```diff
@@ -1496,6 +1496,10 @@ public static void catchAssertError(Throwable e) {
         Assert.fail(e.getMessage());
     }
 
+    public static void BrowserRefresh(){
```
Comment (snaphray) [inline src/test/java/utilities/KeywordUtil.java:1502]: add proper comment for method functionality

File: src/test/java/step_definitions/SalesForceMarketing.java [hunk 3]
```diff
@@ -128,6 +129,61 @@ public void click_contactTab() throws InterruptedException {
...
+
+    @Then("user create new contact in Contact")
+    public void createNewContact() throws InterruptedException {
+        CampaignUtil.createConatctInConatctTab( dataMap.get("ContactLastName"), KeywordUtil.generateRandomString(4) + "@gmail.com");
+        CampaignUtil.verificationMessage();
+        KeywordUtil.takeScreenshotAndAttachInReport();
+    }
+
+    @Then("user create new opportunity in Contact")
+    public void createOpportunityInConatct() throws InterruptedException {
```
Comment (snaphray) [inline src/test/java/step_definitions/SalesForceMarketing.java:164]: Please always check the spelling, contact spelling is not correct, review your code 2-3 times before raising PR

File: src/test/java/pageobjects/SalesforceObjects/CampaignObject.java [hunk 4]
```diff
@@ -74,11 +107,14 @@ public static By clickParentLink(String campaignName){
     public static By showActions(String campaignName){
         return By.xpath("//table[@role=\"grid\"]//tr//th[@scope=\"row\"]//a[text()='"+campaignName+"']//ancestor::th//following-sibling::td[8]//a//span[text()='Show Actions']");
     }
-    public static WebElement select_Tab(String tabname){
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/CampaignObject.java:77]: is method select_tab not required anymore?

File: src/test/java/SalesforceModules/OppurtunitiesUtil.java [hunk 5]
```diff
@@ -261,71 +167,35 @@ public static void verifyDocumentGenerated(String logStep){
...
+        KeywordUtil.click(OppurtunitiesObject.saveClosedWon,logStep);
     }
 
-    /**
-     * Enter createOpportunity
-     *
-     *
-     * @param name the name
-     * @param amount the amount
-     * @param stage the stage
```
Comment (snaphray) [inline src/test/java/SalesforceModules/OppurtunitiesUtil.java]: Please take check this file, you have removed almost all methods description comments, review your code before raising the PR

**Review by snaphray — CHANGES_REQUESTED** [review]
Please resolve the review comments

File: src/test/java/pageobjects/SalesforceObjects/CampaignObject.java (hunk 4 above)
Comment (ishrartx) [reply src/test/java/pageobjects/SalesforceObjects/CampaignObject.java:77]: no it is not required

**Review by snaphray — APPROVED** [review]
Code looks fine

**Pull request merged**